src/dash-table/derived/header/wrappers.tsx

```diff
@@ -1,47 +1,85 @@
 import * as R from 'ramda';
 import React from 'react';
 
-import {memoizeOneFactory} from 'core/memoizer';
+import memoizerCache from 'core/cache/memoizer';
+import derivedCellEventHandlerProps, {
+    Handler
+} from 'dash-table/derived/cell/eventHandlerProps';
+import {
+    ColumnId,
+    Columns,
+    ICellFactoryProps
+} from 'dash-table/components/Table/props';
 
-import {Columns} from 'dash-table/components/Table/props';
+class Wrappers {
+    constructor(
+        propsFn: () => ICellFactoryProps,
+        private readonly handlers = derivedCellEventHandlerProps(propsFn)
+    ) {}
 
-function getter(
-    columns: Columns,
-    labelsAndIndices: R.KeyValuePair<any[], number[]>[],
-    mergeHeaders: boolean
-): JSX.Element[][] {
-    return R.map(([labels, indices]) => {
-        return R.addIndex<number, JSX.Element>(R.map)((columnIndex, index) => {
-            const column = columns[columnIndex];
+    get = (
+        columns: Columns,
+        labelsAndIndices: R.KeyValuePair<any[], number[]>[],
+        mergeHeaders: boolean
+    ): JSX.Element[][] =>
+        labelsAndIndices.map(([labels, indices], rowIndex) =>
```

Comment:
Clean up. TypeScript + R.map + R.addIndex don't make a great mix, requiring explicit typing of everything in most cases. The native implementation is easier to read and is also typed.